packages/panw_cortex_xdr/data_stream/alerts/elasticsearch/ingest_pipeline/default.yml

```
@@ -19,6 +19,8 @@ processors:
   - json:
       field: event.original
       target_field: panw_cortex.xdr
+  - drop:
+      if: ctx.panw_cortex?.xdr?.reply?.result_count != null && ctx.panw_cortex?.xdr?.reply?.result_count == 0
```

Can we elide the null check?
```suggestion
      if: ctx.panw_cortex?.xdr?.reply?.result_count == 0
```

---

Will this not be an issue if the field doesn't exist?

---

That's why it's a question, it's worth checking; the reasoning is that if the field is not present at any level it resolves to approximately (but not actually) `if: null == 0` which is false. This can be checked in painless lab with 
```
HashMap a = new HashMap();
a?.a?.a == 0;
```
which reports `false`.